lib/models/patient/patient.js

```diff
@@ -90,9 +90,11 @@ module.exports = function (gfs) {
     // IMPORTANT: if reset, all patient avatars should be deleted as well
     PatientSchema.plugin(autoIncrementId, { slug: "patientId" }); // auto incrementing IDs
 
-    PatientSchema.virtual("sharesCount").get(function() {
-      return this.shares.length;
-    });
+    if (process.env.NODE_ENV !== "test") {
+      PatientSchema.virtual("sharesCount").get(function() {
```

Comment:
I do not understand this. If this is not needed in a test why are we conditionally removing this for tests. Is !== supposed to be ===?